### ISSUE TITLE
Start activity from background message

### ISSUE THREAD

**Issue by rvarkonyi** [bkdev98/react-native-incoming-call#2]
Hello, 

thanks for this module, it's the only one I was able to use successfully. I'm using Firebase to send a notification to the app, that part is working fine, I can see the incoming message (foreground / background) etc. I'm able to start IncomingCall from foreground but it doesn't want to trigger from the background handler. This is my code:

```// Register background handler
messaging().setBackgroundMessageHandler(async remoteMessage => {      
  console.log('Message handled in the background!', remoteMessage); 
  // Display incoming call activity
  IncomingCall.display('123123', 'caller ID');  
});
```
I can see the log entry for the incoming message so the handler is correctly set up I think. But the call doesn't appear on the device. I might be missing something obvious, could you please help?

Thanks

**Comment by bkdev98** [comment]
Hi. This module is still working in progress and I know this part is not completed. I will fix it in the next few days.

**Comment by bkdev98** [comment]
I just see a fork that seems to fix this issue, you should check this https://github.com/dotlabdev/react-native-incoming-call

**Comment by rvarkonyi** [comment]
Thanks, I was able to get it working using the fork!

**Comment by bkdev98** [comment]
Fixed in v1.2.0
